### PR TITLE
Fixed QUnit tests - api select and content-editable api-actions (part of #471)

### PR DESCRIPTION
**Pull request by georgiy-abbasov**
\cc @AlexanderMoskovkin, @helen-dikareva, @VasilyStrelyaev 
FPR (all previous remarks [here](https://github.com/DevExpress/testcafe/pull/539))

### CODE REVIEW

File: test/client/before-test.js [hunk 1]
```diff
@@ -77,4 +77,34 @@
     window.getCrossDomainPageUrl = function (filePath) {
         return window.QUnitGlobals.crossDomainHostname + window.QUnitGlobals.getResourceUrl(filePath);
     };
+
+    // HACK: IOS browser performs unexpected scrolling in some cases (https://github.com/DevExpress/testcafe/issues/471)
+    // With this hack we allow to set scroll only by a script and prevent native browser scrolling.
```
Comment (VasilyStrelyaev) [inline test/client/before-test.js]:
```
    // HACK: The iOS browser performs unexpected scrolling in some cases (https://github.com/DevExpress/testcafe/issues/471)
    // With this hack, we only allow setting the scroll by a script and prevent native browser scrolling.
```

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Comment by AlexanderMoskovkin** [comment]
so, it seems the target tests are stable. We can review this /cc @helen-dikareva

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by helen-dikareva** [comment]
lgtm

**Pull request merged**